src/book/summary.rs

```diff
@@ -377,6 +377,9 @@ impl<'a> SummaryParser<'a> {
                     items.push(item);
                 }
                 Some(Event::Start(Tag::List(..))) => {
+                    if items.is_empty() {
+                        continue;
```

Comment:
Thanks!  Can you add a comment here explaining why it is doing this?

Reply:
Thanks!
I added a comment.